packages/go/test/fixtures/21-func-conflicts/vercel.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": 2,
+  "builds": [{ "src": "**/*.go", "use": "@vercel/go" }],
+  "probes": [
+    { "path": "/", "mustContain": "from index.go" },
+    { "path": "/dupe-handler.go", "mustContain": "from dupe-handler.go" },
+    { "path": "/other-package.go", "mustContain": "from other-package.go" },
+    { "path": "/sub/one.go", "mustContain": "from /sub/one.go" },
```

Comment:
Out of curiosity, what is this `probes` thing? The docs describe something else and this looks like a Jest thing, but it's obviously in the `vercel.json` file.

Secondly, is it just doing a string match on an absolute path? In other words, it's not actually treating `/sub/one.go` as absolute.

Reply:
Probes are a way to write some simple request/response tests against a deployed fixture.

The `path` is relative. It's appended on to the deployment url.

The logic for this is in: https://github.com/vercel/vercel/blob/9c66ebc536c4acf6ca395aa24fe43100582a817f/test/lib/deployment/test-deployment.js#L34

Note that we recently made a change to allow `probes.json` to contain these instead of putting them in `now.json` or `vercel.json`, which makes it easier to deploy a fixture manually.

Reply:
@EndangeredMassa @cb1kenobi FYI, you can remove `vercel.json` for most tests now and instead use `probes.json`

See https://github.com/vercel/vercel/pull/8279

Reply:
Indeed! I'll switch that over in my follow-up test suite updates PR.